salt/modules/redismod.py

```diff
@@ -69,6 +69,17 @@ def _sconnect(host=None, port=None, password=None):
     return redis.StrictRedis(host, port, password=password)
 
 
+def _get_kwargs_opts(**options):
```

Comment:
@mirceaulinic I'm sure you're busy this week but when you get a moment if you could let me know if you want to come back and make this change, please let me know. Thanks much.